### PR TITLE
#169153746 CRUD Ratings for a single engineer.

### PR DESCRIPTION
**Pull request by gik-hub**
#### What does this PR do?
- This PR adds the endpoint to rate a single engineer
#### Description of Task to be completed?
- add endpoint to add a rate for a single engineer
- add endpoint to update the rate of an engineer
- add endpoint to get the ratings of an engineer
#### Any background context you want to provide?
N/A
#### What are the relevant pivotal tracker stories?
[#169153746](https://www.pivotaltracker.com/story/show/169153746)
#### Screenshots (if appropriate)
image
image
image

#### Questions:
N/A

### CODE REVIEW

File: src/services/ratingService.js [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+            // });
+              //[sequelize.fn('count', sequelize.json('communication.rate'),'communication'), 'communications'],
+
+
+              //[sequelize.json('quality.rate'),'quality'],
+                // [sequelize.json('quantity.rate'),'quantity'],
+                // [sequelize.json('initiative.rate'),'initiative'],
+                // [sequelize.json('communication.rate'),'communication'],
+                // [sequelize.json('professionalism.rate'),'professionalism'],
+                // [sequelize.json('integration.rate'),'integration']
```
Comment (drayzii) [inline src/services/ratingService.js]: Can you delete these comments if they are unnecessary

**Review by drayzii — CHANGES_REQUESTED** [review]
Great job, but would it be possible to add a check preventing an engineer being graded twice in the same week

**Review by Jordybastien — COMMENTED** [review]
You did a great work here, however, if you could implement the feedbacks @drayzii left, that would be great

**Comment by gik-hub** [comment]
Sure, we are gonna work on it ASAP

File: src/services/ratingService.js (hunk 1 above)
Comment (willywax) [reply src/services/ratingService.js]: Will delete the comments. About an engineer being  graded twice, According to Denis an LF can rate an engineer even for 5 weeks at once. So no check is needed.

**Review by Jordybastien — APPROVED** [review]

File: src/services/ratingService.js (hunk 1 above)
Comment (drayzii) [reply src/services/ratingService.js]: @willywax cool, but then how will we link grades to a week? should we add it to the ratings table

Comment (drayzii) [reply src/services/ratingService.js]: For the LF to add the week

**Review by drayzii — APPROVED** [review]
LGTM

File: src/services/ratingService.js (hunk 1 above)
Comment (willywax) [reply src/services/ratingService.js]: @drayzii Nice observation. But according to the PO we don't provide ratings weekly. So it's possible to provide rating to an Engineer even 4 weeks in a single day.

**Pull request merged**